metrics/core/types.go

```diff
@@ -119,6 +118,8 @@ func (this *LabeledMetric) GetValue() interface{} {
 }
 
 type MetricSet struct {
+	CreateTime     time.Time
+	ScrapeTime     time.Time
```

Comment:
Scrape time is (at least now) used only for converting rate metrics to cumulative. Do you have any cumulative metric that is scraped at a very different time? This is an internal structure that can be changed in the future. Right now I'm trying to reduce a size of a single metric as we are adding couple more of them.